syntaxes/Asciidoctor.json

```
@@ -1998,7 +1998,7 @@
               "include": "#tables-includes"
             }
           ],
-          "end": "^(\\1)$",
+          "end": "^(\\1\\s*)$",
```

I guess spaces should be outside of the group?

```suggestion
          "end": "^(\\1)\\s*$",
```